src/toil/job.py

```diff
@@ -609,14 +609,14 @@ def cores(self, val: ParseableDivisibleResource) -> None:
         self._requirementOverrides["cores"] = Requirer._parseResource("cores", val)
 
     @property
-    def preemptable(self) -> bool:
-        """Whether a preemptable node is permitted, or a nonpreemptable one is required."""
-        return cast(bool, self._fetchRequirement("preemptable"))
-
-    @preemptable.setter
-    def preemptable(self, val: ParseableFlag) -> None:
-        self._requirementOverrides["preemptable"] = Requirer._parseResource(
-            "preemptable", val
+    def preemptible(self) -> bool:
```

Comment:
Since this is public as well, is there a way to make this backward compatible? Or perhaps this would be a major release with many API changes?